src/components/common/YourCompany/YourCompany.vue

```diff
@@ -451,32 +450,26 @@ export default class YourCompany extends Mixins(
   /** The recognition/founding (aka effective or start date) datetime. */
   get recognitionDateTime (): string {
     if (this.isBenIaCorrectionFiling) {
-      if (this.getOriginalEffectiveDateTime) {
-        return (this.apiToPacificDateLong(this.getOriginalEffectiveDateTime))
+      if (this.getCorrectedFilingDate) {
```

Comment:
The old getter looked in the corrected IA, whereas this getter look at the correction object property. They are the same value.